regression-tests/test-results/gcc-14-c++2b/pure2-default-arguments.cpp.execution

```diff
@@ -1,4 +1,4 @@
 calling: int main(int, char**)
 012
-a newer compiler
+an older compiler
```

Comment:
Is this intended? 14 is the most recent version of GCC  released only a few months back.

Reply:
I think this is a bug in https://github.com/hsutter/cppfront/blob/2e2359772a13fb707d039f9289deaf4683288ffb/include/cpp2util.h#L367 and they should be `>=` rather than `>`.

Reply:
It does smell like it. @hsutter do you confirm, should I fix it in this PR?

Reply:
This test does not test whether gcc-14 is modern but whether you can test its version at compile time.

Reply:
I agree, but it is still a bit strange to see a very recent compiler marked as old.
To me the name `gcc_clang_msvc_min_versions` suggest that if the actual version is equal to the value stored internally the version should be accepted.